test/test_client.py

```diff
@@ -29,6 +30,20 @@ def setUp(self):
         self._response = Mock()
         self._response.status_code = 200
 
+    def test_send_version_with_internal_metrics(self):
+        no_registry = registry.WavefrontSdkMetricsRegistry(
+            wf_metric_sender=None)
+        with patch.object(
+                registry,
+                'WavefrontSdkMetricsRegistry',
+                return_value=no_registry) as mock_registry:
+            WavefrontClient(
+                'no_server',
+                'no_token',
+                flush_interval_seconds=86400,
+                enable_internal_metrics=True)
+        self.assertTrue('version' in mock_registry.call_args[1]['tags'])
```

Comment:
I assume we have an actual value for the `version` tag? If so, can we assert that it matches a semver regex? For example: `^v\d+\.\d+\.\d+$`

Reply:
While the semver can be fetched when I run the unit tests locally, fetching the semver on github fails so on github version resolves to **unknown**.  Therefore I tweaked the regex you provided to allow for either **unknown** or a semver.